controller/src/main/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessors.java

```diff
@@ -193,12 +194,23 @@ public boolean isMetadataServiceConnected() {
         return  checkpointStore.isHealthy();
      }
 
+    /**
+     * Get bootstrap completed status.
+     *
+     * @return true if bootstrapCompleted is set to true.
+     */
+    public boolean isBootstrapCompleted() {
+        return this.bootstrapCompleted.get();
+    }
+
     /**
      * Get the health status.
      *
      * @return true if zookeeper is connected and bootstrap is completed.
      */
     public boolean isReady() {
+        log.info("inside CEP isReady method, values of isMetadataServiceConnected is {} and isBootstrapCompleted is {}",
```

Comment:
I don't know if this is the best message possible for an `info` level, looks like more of a debug message. Also, please log messages start by capital.

Reply:
The log text does not look like a production level log. You can choose to remove it altogether or replace the message with a better one and change the level to debug.

Reply:
This was mainly to check for the manual testing. Changed the text and added `log.debug`